ush/config_defaults.yaml

```diff
@@ -901,7 +1050,7 @@ task_make_grid:
   # 
   #-----------------------------------------------------------------------
   # 
-  GRID_DIR: ""
+  GRID_DIR: '{{ [workflow.EXPTDIR, "grid"]|path_join if workflow_switches.RUN_TASK_MAKE_GRID else [platform.DOMAIN_PREGEN_BASEDIR, workflow.PREDEF_GRID_NAME]|path_join}}'
```

Comment:
I am curious. Does the yaml-xml work needs all variables to be defined in the config_defaults.yaml and not in setup.py?

Reply:
It does not. This is me attempting to get all defaults set in the same place. Before this there wasn't a clear distinction on where they were set. Some in setup, others in config_defaults.

A few that I moved to config_defaults.yaml were a bit borderline on where they should be. This was definitely one.